src/tools/webui/README.md

```diff
@@ -4,24 +4,9 @@ The client for elektra-web, built using
 [React](https://facebook.github.io/react/) and [Redux](http://redux.js.org/). It
 accesses the API that [`webd`](../webd/) provides.
 
-## Installation
+## Installation and Usage
 
-- first, install [libelektra](http://libelektra.org/) with the [`yajl` plugin](http://tree.libelektra.org/src/plugins/yajl/)
-- now run `npm install`
-
-## Running
-
-Start the client in **production** mode via:
-
-```
-npm start
-```
-
-Alternatively, you can use **development** mode:
-
-```
-npm run start:dev
-```
+You can find a comprehensive tutorial on how to install and use the webui [here.](/doc/tutorials/install-webui.md)
```

Comment:
Did this tutorial work for you?

Reply:
Yes, i used it actually and I think it could save other people some time, since it contains more information than just the yajl plugin info.